__test__/functions/connection/onConnect.test.ts

```diff
@@ -1,9 +1,8 @@
 import * as LambdaTester from 'lambda-tester';
 import { handler } from '../../../src/functions/connection/onConnect';
 import * as userFunctions from '../../../src/dynamodb/userDBService';
-import { response } from '../../../src/utils/responseHelper';
 import { createEvent } from '../functionsTestHelpers';
-import { LambdaResponse } from '../../../src/types/response';
+import { FAKE_CONNECTION_ID1 } from '../../testConstants';
 
 const TEST_CONNECTION_ID = 'test-onConnect-connectionId';
```

Comment:
Should we also add TEST_CONNECTION_ID to testConstant file?

Reply:
kk